src/compile/model.ts

```diff
@@ -269,7 +269,11 @@ export abstract class Model {
     // Increase ref count. This is critical because otherwise we won't create a data source.
     // We also increase the ref counts on OutputNode.getSource() calls.
     const refCounts = this.component.data.outputNodeRefCounts;
-    refCounts[fullName]++;
+    if (!(fullName in refCounts)) {
```

Comment:
If you wanna do one line, you can do

refCounts[fullName] = (refCounts[fullName] || 0) + 1;